numerics/cbrt_test.cpp

```diff
@@ -38,6 +43,77 @@ class CubeRootTest : public ::testing::Test {
       : quiet_dead_beef_(FromBits(0x7FF8'0000'DEAD'BEEF)),
         signaling_dead_beef_(FromBits(0x7FF0'0000'DEAD'BEEF)) {}
 
+  struct RoundedReal {
```

Comment:
"Real" is not a word in C++.  I'd prefer `RoundedDouble`.

Reply:
Real is a real word in the real world though. We round a real, that is, an element of ℝ (which in this case is the cube root of an element of binary64) to binary64 in any of three ways.
Rounding a `double` makes little sense, it is in `binary64` already.